<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>419</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Фамилия&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Личный номер</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Дата рождения</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Образование</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Должность</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Профессия</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Номер подразделения</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Семейное положение</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Количество детей</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Название подразделения</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Руководитель</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="surname"/>
        </item>
        <item>
         <widget class="QLineEdit" name="name"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lastName"/>
        </item>
        <item>
         <widget class="QLineEdit" name="personalNumber"/>
        </item>
        <item>
         <widget class="QLineEdit" name="birthday"/>
        </item>
        <item>
         <widget class="QLineEdit" name="education"/>
        </item>
        <item>
         <widget class="QLineEdit" name="post"/>
        </item>
        <item>
         <widget class="QLineEdit" name="profession">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="divisionNumber"/>
        </item>
        <item>
         <widget class="QLineEdit" name="marritalStatus">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numberOfChildren"/>
        </item>
        <item>
         <widget class="QComboBox" name="divisionName"/>
        </item>
        <item>
         <widget class="QLineEdit" name="director"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QDialogButtonBox" name="okCancelButton">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
